src/functions/keycloak/list-users/handler.ts

```diff
@@ -1,54 +1,53 @@
 import { middyfy } from "@libs/lambda";
 import type { APIGatewayProxyHandler } from "aws-lambda";
-import { error } from "console";
 import { CreateKeycloakApiService, type KeycloakApiService } from "src/apis/keycloak-api-service";
 
 /**
  * Response schema for lambda
  */
 interface Response {
-    id: string;
-    firstName: string;
-    lastName: string;
-    email: string;
-    isActive: boolean;
-    severaGuid: string;
-    forecastId: number;
+	id: string;
+	firstName: string;
+	lastName: string;
+	email: string;
+	isActive: boolean;
+	severaGuid: string;
+	forecastId: number;
 }
 
 /**
  * Lambda for listing users
  */
 const listUsersHandler: APIGatewayProxyHandler = async () => {
-    try {
-        const api = CreateKeycloakApiService();
-        
-        const users: Response[] = await api.getUsers().then((users) => {
-            return users.map(user => {
-                const responseUser: Response = {
-                    id: user.id,
-                    firstName: user.firstName,
-                    lastName: user.lastName,
-                    email: user.email,
-                    isActive: user.isActive,
-                    severaGuid: user.severaGuid,
-                    forecastId: user.forecastId
-                };
-                return responseUser;
-            });
-        });
+  try {
+	  const api = CreateKeycloakApiService();
+		
+	  const users: Response[] = await api.getUsers().then((users) => {
```

Comment:
Typically we use async await OR .then.
And generally we use async await instead of .then.
So I'd recommed changing this to use just async await and not the .then.

This can be done by awaiting the getUsers() and assigning it to a named variable and then mapping as required,

Reply:
Made changes to Iiros code. Now it has async await structure.